spec/uap_clj/common_spec.clj

```diff
@@ -1,3 +1,27 @@
-(ns uap-clj.common-spec)
+(ns uap-clj.common-spec
+  (:require [clojure.java.io :as io :refer [resource]]
```

Comment:
another nit: you don't need the refer since you only refer to `resource` via the `io` alias (i.e. `io/resource`)